### ISSUE TITLE
a11y: description lists

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#786]
**What Should Happen**
Metadata (e.g. at the top of reports, in the View Data Collisions section) should be marked up using description lists.

**JIRA Issues**
- [x] MOVE-165
- [x] MOVE-150
- [x] MOVE-121
- [ ] MOVE-115
- [x] MOVE-88
- [x] MOVE-16
- [ ] MOVE-15
